### DISCUSSION TITLE
How do I connect a ethernet usb adapter to a wyze cam v3?

### DISCUSSION THREAD

**Discussion by johnvpr** [gtxaspec/wz_mini_hacks#681] [General]
hi guys, 

How do I connect a ethernet usb adapter to a wyze cam v3?

Thanks

**Comment by bjs-pdx** [comment]
It needs to connect to the little cable coming out of the camera.  Standard USB adapters don't work.  You need something with the correct connector (micro?mini? can't remember which offhand.)  Did you read the post on working adapters?

https://github.com/gtxaspec/wz_mini_hacks/discussions/73
